source/sink-connector/configuration-properties/write-strategies.txt

```diff
@@ -23,6 +23,15 @@ configuration:
 
    writemodel.strategy=<a writemodel strategy>
 
+You can also select a delete write model strategy to specify how the
```

Comment:
Question:
Which term is correct: "delete writemodel" or "delete write model"?
And, since these strategies can be applied to both, should it include that qualifier? I think that could be confusing to refer to the same thing in two ways without explaining the connection.